README.md

```diff
@@ -76,4 +77,16 @@ By default, installing a package with the `—save` option, npm saves the packag
 npm config set save-exact
 ```
 
+[↑ back to Table of Contents](#table-of-contents)
+
+## Write atomic tasks
+
+### Why?
+
+Convoluted tasks are harder to debug and understand. 
+
+### How?
+
+Each task should be only responsible for one function. If a specific task requires some extra complexity like cleaning a directory and then generating icons, just separate the clean task from generating icons. 
+
```

Comment:
I would consider illustrating this with actual scripts. And maybe using a more familiair set of tasks, like "optimise svg icons", "generate icon sprites" , ... ?